to_multi_var

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOT_DIRECTORY="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )" )"
```

Comment:
You changed directory and then used `pwd`. `readlink -f` is a way to resolve without changing directory.